docs/dietpi_tools.md

```diff
@@ -545,7 +545,7 @@ DietPi Survey allows the DietPi project to obtain general information regarding
     which displays the status like
 
     ```
-    root@dietpi:~# grep 'SURVEY_OPTED_IN' /boot/dietpi.txt
+    # grep 'SURVEY_OPTED_IN' /boot/dietpi.txt
```

Comment:
Shall we also delete `root@dietpi` in line 523?
`root@dietpi:~# dietpi-software list | grep ' =2'`

Reply:
I'd say so. Prefixing executed commands with `#` or `$` in general or to separate from output is quite common, as those are the root and non-root default PS1 (command prompt) prefixes across shells.

Reply:
I will go through all /boot/dietpi scripts, search for phpBB leftovers and do PRs for this.